packages/avatar/project.json

```
@@ -46,7 +46,14 @@
       "executor": "ngx-deploy-npm:deploy",
       "options": {
         "access": "public",
-        "buildTarget": "production"
+        "distFolderPath": "dist/packages/avatar"
+      },
+      "dependsOn": ["pre-npm-deploy-build-production"]
+    },
+    "pre-npm-deploy-build-production": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "nx run avatar:build:production"
```

I see that the default mode of the build is `production`, so another change that we could implement is to do the following to make it more streamlined.

```suggestion
         "distFolderPath": "dist/packages/avatar"
       },
       "dependsOn": ["build"]
     }
```